api/web3server_test.go

```diff
@@ -313,35 +306,30 @@ func TestEstimateGas(t *testing.T) {
 		},
 	}
 
-	for _, v := range testData {
-		ret, err := svr.web3Server.estimateGas(v.input)
-		require.NoError(err)
-		require.Equal(ret, uint64ToHex(v.result))
+	for i, v := range testData {
+		t.Run(fmt.Sprintf("%d-%d", i, len(testData)-1), func(t *testing.T) {
```

Comment:
IMO we should not use `t.Run` here inside for loop? only 3 lines of code, not worth it

`t.Run` is meant to run the entire test with a different config, like
```
testIndexer := func(kvStore db.KVStore, t *testing.T) {
   // 100 lines of test code
}
t.Run("in memory", func(t *testing.T) {
	testIndexer(db.NewMemKVStore(), t)
})
t.Run("db files", func(t *testing.T) {
	testIndexer(db.NewBoltDB(), t)
})
```